parastell/invessel_build.py

```diff
@@ -226,39 +217,71 @@ def calculate_loci(self):
 
         [surface.calculate_loci() for surface in self.Surfaces.values()]
 
+    def _check_edge(self, edge):
```

Comment:
Functions returning booleans may be better with a name that indicates the outcome, like `_is_near_end`